### DISCUSSION TITLE
En busca de una persona para el hackathon

### DISCUSSION THREAD

**Discussion by igutisan** [Launch-X-Latam/MisionFrontEnd#505] [Team Posts]
Hola, quien guste hacer parte de mi grupo pueden responder este mensaje con su correo de teams. Gracias

**Comment by laputorres** [comment]
Hola yo puedo, te mando mi correo lx14852@innovaccion.mx

**Comment by CondeCesarSA** [comment]
Hola mi correo de team es  ***@***.***

El mar, 8 nov 2022 a la(s) 16:04, laputorres ***@***.***)
escribió:

> Hola yo puedo, te mando mi correo ***@***.***
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/Launch-X-Latam/teams/mision-backend/discussions/34/comments/1>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AJ42XRGNKEKFPKGOG5QQ2JDWHKP3BANCNFSM6AAAAAAR2UU6AQ>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>


-- 
*Atte : Conde César Santiago Andrés.*

**Comment by TheValentin** [comment]
hola puedo entrar a su grupo de hackaton, mi correo es: lx06687@innovaccion.mx, me llamo andy valentin soy de Perú

**Comment by jlnunezobando** [comment]
Hola puedo entrar a tu grupo mi correo es: lx08651@innovaccion.mx
Jorge Nuñez de colombia.

**Comment by CAMILOITT** [comment]
hola mi nombre es Camilo estoy interesado en entrar en tu grupo 
correo : lx11002@innovaccion.mx

**Comment by Hernanjas400** [comment]
Hola mi nombre es Hernan Soy de Argentina me gustaria participar en su grupo Correo: lx06198@innovaccion.mx

**Comment by stvnmc** [comment]
stvnmc
github
.
teams
***@***.*** ***@***.***>
